src/feed.tsx

```diff
@@ -8,18 +9,64 @@ export interface Properties {
   app: string;
 }
 
-export class Feed extends React.Component<Properties> {
+export interface State {
+  feed: FeedItemModel[];
+  hasMore?: boolean;
+  pageNumber?: number;
```

Comment:
values in state shouldn't need to be optional. let's just initialize them with expected values. you can either do that in the constructor, or using an instance variable:
`state = { feed: [], hasMore: false, pageNumber: 0 };`